templates/two-columns/two-columns.js

```diff
@@ -0,0 +1,60 @@
+import { loadCSS } from '../../scripts/lib-franklin.js';
+
+/**
+ * Checks if the ratio is different from 50-50 and applies the ratio to its column respectively
+ * @param {HTMLElement} columnOne - first column section
+ * @param {HTMLElement} columnTwo - second column section
+ */
+function setRatio(columnOne, columnTwo) {
+  const ratio = document.querySelector('meta[name="column-ratio"]').content;
+  if (ratio) {
+    const isDefault = ratio.includes('50');
+    if (!isDefault) {
+      const [firstRatio, secondRatio] = ratio.split('-');
+      const root = document.querySelector(':root');
+
+      root.style.setProperty('--first-ratio', `${firstRatio}%`);
+      root.style.setProperty('--second-ratio', `${secondRatio}%`);
+
+      columnOne.classList.add('first-column');
+      columnTwo.classList.add('second-column');
```

Comment:
these two lines should be outside of the `isDefault` if, otherwise the columns will never get applied, right?

Reply:
By default, the ratio is 50-50, so these two classes are not needed. unless the ratio is different from 50, then those 2 classes have to be added

Reply:
I tried the fall back and it seems like it is not working: https://255-marketo-form-cannot-be-embedded--cn-website--netcentric.hlx.page/drafts/jlledo/two-columns-demo

Reply:
fixed